model/simulationtests/ghx_horizontal_trench.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+require 'openstudio'
+require_relative 'lib/baseline_model'
+
+model = BaselineModel.new
+
+# No zones, just a LoadProfile:Plant
+# This is unfortunate but the only E+ example file that uses it is PlantHorizontalGroundHX.idf
+# and it has just a LoadProfile:Plant, and nothing autosized
+# We couldn't get this file to not throw a Plant run away temperature so we're
+# matching the E+ test instead
+
+# add design days to the model (Chicago)
+model.add_design_days
+
+USE_KUSUDA = true
+USE_XING = false
+USE_PIPE_INDOOR = false
+
+raise "Cannot use XING on 3.5.1 and below" if USE_XING and Gem::Version.new(OpenStudio.openStudioVersion) <= Gem::Version.new('3.5.1')
```

Comment:
Ok so for now I have left these switches, until we are satisfied with what we're going.

I suppose eventually after we're done testing we need to make two ruby tests: A kusuda one, and a Xing one.

The kusuda should be run at 3.5.1, and the osm saved, and results commited.

the Xing requires the next official version so only the ruby one should be added.